src/main/java/com/cloudbees/jenkins/support/impl/JenkinsLogs.java

```diff
@@ -126,6 +126,15 @@ private void addOtherMasterLogs(Container result) {
                 }
             }
         }
+        File taskLogs = new File(logs, "tasks");
+        if (taskLogs.exists() && taskLogs.isDirectory()) {
```

Comment:
Redundant—`isDirectory` covers all relevant cases.

Actually it is not even necessary to check that here, or in the `logs` block above: if the directory does not exist, `listFiles` will return null, which we are already checking for.